tests/log/test_log_entry_added.py

```diff
@@ -353,10 +354,12 @@ async def test_exceptionThrown_logEntryAddedEventEmitted(
             "timestamp": ANY_TIMESTAMP,
             "stackTrace": {
                 "callFrames": [{
-                    "url": "",
+                    "url": url,
                     "functionName": "",
                     "lineNumber": 0,
-                    "columnNumber": 14
+                    # Column number is a magical constant. It depends on the
+                    # html fixture wrapping content in document tag.
+                    "columnNumber": 127
```

Comment:
How about we use `ANY_NUMBER`?

Reply:
This is expected to be a constant, so let's keep it